Source/Scene/Cesium3DTilesetAsyncTraversal.js

```diff
@@ -0,0 +1,141 @@
+define([
+        '../Core/Intersect',
+        '../Core/ManagedArray',
+        './Cesium3DTileRefine'
+    ], function(
+        Intersect,
+        ManagedArray,
+        Cesium3DTileRefine) {
+    'use strict';
+
+    /**
+     * Traversal that loads all leaves that intersect the camera frustum.
+     * Used to determine ray-tileset intersections during a pickFromRayMostDetailed call.
+     *
+     * @private
+     */
+    function Cesium3DTilesetAsyncTraversal() {
+    }
+
+    var asyncTraversal = {
+        stack : new ManagedArray(),
+        stackMaximumLength : 0
+    };
+
+    Cesium3DTilesetAsyncTraversal.selectTiles = function(tileset, frameState) {
+        tileset._selectedTiles.length = 0;
+        tileset._requestedTiles.length = 0;
+        tileset._hasMixedContent = false;
+        var minimumGeometricError = 0;
```

Comment:
Should this be a variable?  It is never written to.  Is it clearer to just use `0.0` (it's a float) twice below?